tempest/api/workloadmgr/workload_reassign/test_workload_reassign.py

```diff
@@ -490,4 +490,235 @@ def test_05_workload_reassign(self):
+                [vm_id], tvaultconf.parallel)
+            LOG.debug("Workload ID: " + str(workload_id))
+
+            if(workload_id is not None):
+                self.wait_for_workload_tobe_available(workload_id)
+                if(self.getWorkloadStatus(workload_id) == "available"):
+                    reporting.add_test_step("Create workload", tvaultconf.PASS)
+                else:
+                    LOG.error("Failed to get workload status as available for workload ID: " + str(workload_id))
+                    raise Exception("Create workload")
+            else:
+                LOG.error("Failed to create workload.")
+                raise Exception("Create workload")
+
+            #take a full snapshot of it.
+            snapshot_id = self.create_snapshot(workload_id, is_full=True)
+
+            snapshot_status = self.getSnapshotStatus(workload_id, snapshot_id)
+
+            if (snapshot_status == "available"):
+                LOG.debug("Full snapshot created.")
+                reporting.add_test_step("Create full snapshot", tvaultconf.PASS)
+            else:
+                LOG.error("Full snapshot creation failed.")
+                raise Exception("Create full snapshot")
+
+
+            #use temp user and temp project for workload reassign.
+            rc = self.workload_reassign(self.temp_tenant_id, workload_id, self.temp_user_id)
+            if rc == 0:
+                LOG.debug("Workload reassign to temp project with temp user is passed")
+                reporting.add_test_step(
+                    "Workload reassign to temp project with temp user", tvaultconf.PASS)
+            else:
+                LOG.error("Workload reassign to temp project with temp user is failed")
+                raise Exception("Workload reassign to temp project with temp user")
+
+            #delete the current created project.
+            resp = self.delete_project(self.temp_tenant_id)
+            if resp:
+                LOG.debug(f"Project {self.temp_tenant_id} deleted successfully")
+                reporting.add_test_step(
+                    "Project deletion", tvaultconf.PASS)
+            else:
+                LOG.error(f"Project {self.temp_user_id} is not deleted successfully")
+                raise Exception("Project deletion")
+
+            #use same deleted temp project for workload reassign.
+            rc = self.workload_reassign(self.temp_tenant_id, workload_id, self.temp_user_id)
+            if rc == 0:
+                LOG.debug("Workload reassign with same temp project and temp user is successful. Test case failed.")
+                raise Exception("Workload reassign with temp project and temp user")
+            else:
+                LOG.error("Workload reassign with same temp project and temp user is failed. Test case Passed")
+                reporting.add_test_step(
+                    "Workload reassign with temp project and temp user", tvaultconf.PASS)
```

Comment:
workload reassign fails in this case, hence need to add similar statement in reporting with result as "Pass".
EX => workload reassign on temp deleted project failed => Result : Pass